cli/cli_app.go

```diff
@@ -61,6 +62,14 @@ func CreateTerragruntCli(version string) *cli.App {
    Moreover, for the apply and destroy commands, Terragrunt will first try to acquire a lock using DynamoDB. For
    documentation, see https://github.com/gruntwork-io/terragrunt/.`
 
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "config",
+			Value: os.Getenv("TERRAGRUNT_CONFIG"),
```

Comment:
I agree that would be nice from a consistency standpoint, but that sort of command would conflict with the underlying terraform command. The `terraform apply` command accepts a path to a saved plan file, so if someone calls `terragrunt apply foo`, we wouldn't know if foo is a Terragrunt config file or a Terraform plan file.